Overview.src.html

```diff
@@ -3391,21 +3456,23 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
   <dl class=switch>
    <dt><code data-anolis-spec=fileapi>Blob</code>
    <dd>
-    <p>Push a copy of <var>object</var>'s contents to <var>stream</var>.
+    <p><span title=concept-enqueue-ReadableStream>Enqueue</span> a <code>Uint8Array</code> object
```

Comment:
Read _object_ in parallel:
 - Whenever one or more bytes are read, let _bytes_be the read and enqueue a Uint8Array object wrapping an ArrayBuffer containing _bytes_ to _stream_.
  - If an exception is thrown, error _stream_ and cancel reading.
 - When the reading is done, close _stream_.

"Read _object_" is not defined for FormData, but is it OK?